lib/api_error_handler/error_reporter.rb

```diff
@@ -24,6 +24,11 @@ def report(error, error_id: nil)
 
         extra = error_id ? { error_id: error_id } : {}
         Raven.capture_exception(error, extra: extra)
+      elsif @strategy == :sentry
```

Comment:
This condition will not trigger because line 22 is `elsif @strategy == :raven || @strategy == :sentry`. You'll need to update that conditional and then this should work fine.

It looks like the `:sentry` strategy works accidentally at the moment because of this check.

Reply:
ah my bad, thought I removed that check :D